api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
This will be tricky to cherry-pick, since this version will be different in the release branch.

Also I kind of feel like, for the user's sake if they're looking at it on their own apiserver, it should include the git hash and dirty/not dirty marker. But we wouldn't want to check that in, since it changes on every commit. Hm. Now I'm rethinking whether this should really be the binary version like we talked about yesterday.